tests/test_resources.py

```diff
@@ -2189,6 +2190,43 @@ def test_subscribe_multiple_errors(self):
         except e:
             self.fail("Failed subscription did not raise a Validation error")
 
+    def test_subscription_change_proration(self):
```

Comment:
Since The proration settings is not available in the response, I did not think it made sense to make a fixture, then assert that the unit_amount_in_cents was right, since it wouldn't actually be testing proration. Instead, I went with testing the XML was formatted correctly.